docs/UserGuide.md

```diff
@@ -166,15 +165,15 @@ Format `cancel APPT_ID`
 Example:
 * `cancel 202003081000` deletes the appointment happening on 08/03/2020 10am.
 
-### Clearing all entries : `clear`
+### Clearing all patient entries : `clear`
```

Comment:
Seeing that appointment is dependent on the patient, I think that it should be clear all appointment entries instead?

Reply:
Makes sense, I will make the change